cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -1621,16 +1629,19 @@ public void addSubscriptionTags(String[] tagIds) {
 
     public void addSubscriptionTopic(String topicId, CompletionListener completionListener) {
```

Comment:
please add overload for add and remove like this:
public void addSubscriptionTopic(String topicId) {
  addSubscriptionTopic(topicId, null);
}